pacta_data_qa.Rmd

```diff
@@ -0,0 +1,678 @@
+---
+title: "PACTA Analysis - Input Data QA"
+author: "Jacob Kastl"
+date: "`r format(Sys.time(), '%B %Y')`"
+mainfont: GT America
```

Comment:
prefer that we don't mess with fonts on something only used internal and/or unless it really matters... too much inconsistency across platforms, systems, environments

Reply:
We can definitely remove this.

Reply:
Done